configure.py

```diff
@@ -1107,7 +1110,6 @@ def find_headers(repodir, excluded_dirs):
 ]
 
 deps['test/boost/duration_test'] += ['test/lib/exception_utils.cc']
-deps['test/boost/alternator_base64_test'] += ['alternator/base64.cc']
```

Comment:
It need to be removed because it has been added in scylla_tests_generic_dependencies which is already added to alternator_base64_test as it is part of scylla_tests list
```
for t in scylla_tests:
    deps[t] = [t + '.cc']
    if t not in tests_not_using_seastar_test_framework:
        deps[t] += scylla_tests_dependencies
    else:
        deps[t] += scylla_core + idls + scylla_tests_generic_dependencies
```